compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -503,6 +510,89 @@ string t_java_generator::java_override_annotation() {
   return "@Override";
 }
 
+map<string, int> t_java_generator::get_keywords_list() const
+{
+    return java_keywords;
+}
+
+void t_java_generator::init_keywords()
+{
+    java_keywords.clear();
+
+    // according to Oracle docs
+    java_keywords["abstract"] = 1;
```

Comment:
can we just create an array and populate the map from that?

Reply:
sure. in netstd there is a simikliar place that you could handle the same way